### PR TITLE
feat: Add main design elements

### PR DESCRIPTION
**Pull request by luanpotter**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
-      <div class="text-2xl">2D game engine made on top of Flutter</div>
-      <div class="space-y-8 flex flex-col md:space-y-0 md:space-x-8 md:flex-row">
-        <a class="main-button" href="https://docs.flame-engine.org/latest/">Get Started</a>
-        <a class="main-button" href="https://github.com/flame-engine/flame">GitHub</a>
-      </div>
-    </div>
+<div class="flex flex-col items-center space-y-8 text-white pt-16">
+  <img src="assets/images/t-rex.png" />
+  <div class="font-bold text-6xl">
+    Versatile, Open Source & Portable
```
Comment (spydon) [inline index.html:9]: I like "runs everywhere" more 😅

**Review by spydon — COMMENTED** [review]
Looks good, just a tagline comment!

File: index.html (hunk 1 above)
Comment (luanpotter) [inline index.html:9]:
```suggestion
    Versatile, Open Source & Runs Everywhere
```

**Review by spydon — APPROVED** [review]
Lgtm!

**Pull request merged**